SeaBattle/BattleShip.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeaBattle
+{
+    public class BattleShip : BaseShip, IAttack
+    {
+        public BattleShip(string Name, List<Point> points)
+        {
+
+            var Decks = new List<Deck>();
+
+            for (int i = 0; i < points.Count - 1; i++)
+            {
+                Deck a = new Deck();
+                a.Location = points[i];
+                Decks.Add(a);
+                a.State = 1;

```

Comment:
You can make this value as a default, so you could remove this assignment.

Reply:
Fixed.